tests/psychrometrics/t_o.test.js

```diff
@@ -95,7 +95,7 @@ describe("t_o", () => {
   );
 
   it("throws an error if the airSpeed is negative", () => {
-    expect(() => t_o(0, 0, -1, "ISO")).toThrow("airSpeed cannot be negative");
+    expect(() => t_o(0, 0, -1, "ISO")).toThrow("v cannot be negative");
```

Comment:
I think we should have some tests here for the array version of the function - just so we have confidence moving forward that we haven't broken things accidentally (see [Test First](https://www.notion.so/Test-First-6fe7edeb7c424182ba49dc973549bbd0?pvs=4))

Reply:
yeah, good point, I will add them